recipes-support/salt/salt_3000.2.bb

```diff
@@ -0,0 +1,203 @@
+HOMEPAGE = "http://saltstack.com/"
+SECTION = "admin"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=c996f5a78d858a52c894fa3f4bec68c1"
+DEPENDS = "\
```

Comment:
NIT: While we're authoring this file, could you update the tab indent-styling. We're converging on the hanging-tab style:
````
VARIABLE = "\
<tab>foobar \
"
```

Reply:
Can you clarify? Do you want a single tab instead of multiple? Do you want to replace spaces with a tab?

Reply:
Both or either; depending on which variable assignment we're talking about.

Reply:
Ok. That did not do much for my confusion :). I took a pass at it. Take a look and lmk if you would like additional changes.

Reply:
You got it. Looks good.